pyproject.toml

```diff
@@ -54,6 +56,8 @@ ignore = [
     "D401",  # non-imprative mood
     "E266",  # multiple leading hashes for block comment
     "W605",  # invalid escape sequence
+    "E111",  # indentation-with-invalid-multiple
+    "E114",  # indentation-with-invalid-multiple-comment
```

Comment:
For those two added lines, please refer to below comment:
https://github.com/Cockatoo-AI-Org/cockatoo_chain/issues/2#issuecomment-2727217674